lib/skate_web/components/core_components.ex

```diff
@@ -0,0 +1,12 @@
+defmodule SkateWeb.CoreComponents do
+  @moduledoc """
+  Provides core UI components.
+  """
+
+  def cdn_route(conn, path) do
```

Comment:
**suggestion (non-blocking):** I might call this something like `static_content_route` since I feel like that better reflects what it does from the perspective of somebody writing a HEEx page (and whether that links to a CDN or not is transparent to them).